src/sagemaker_training/environment.py

```diff
@@ -359,6 +359,24 @@ def num_cpus():  # type: () -> int
     return multiprocessing.cpu_count()
 
 
+def validate_smddprun():  # type: () -> bool
```

Comment:
1. nit: If you want to add function signature, can try the more idiomatic way:
```
def validate_smddprun() -> bool:
    ...
```

2. `validate_...` is more like a sanity check, and it can raise (failing) or not (success), while what the function actually does is more like `def smddprun_is_enabled() -> bool`

Reply:
I sympathize with both comments; however, the reason I formatted like this was to just be consistent with the rest of the code already commited.  For example, there was already a `validate_smddpmprun` function defined (for the smddpmprun launcher) that uses this signature